### PR TITLE
Dice screen

### PR DESCRIPTION
**Pull request by LeonardoMolleker**
Screenshot_16

A page with a Scaffold, inside has an AppBar with a title, a GridView with images of dices and a bottom navigation bar who displays info.

### CODE REVIEW

**Review by FranciscoGaraventa — COMMENTED** [review]
LGTM!

File: lib/src/pages/dice_page.dart [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          );
+        }),
+      ),
+      bottomNavigationBar: Container(
+        decoration: BoxDecoration(
+            border: Border(
+                top: BorderSide(
+          width: Constants.borderSideWidth,
+          color: Colors.blue,
+        ))),
```
Comment (MarcosCGlobant) [inline lib/src/pages/dice_page.dart]: can you try to do Ctrl + Alt + L here? This part of the code doesn't look aligned, tell me if it works

**Review by MarcosCGlobant — APPROVED** [review]
one alignment comment, beside that LGTM

File: lib/src/pages/dice_page.dart (hunk 1 above)
Comment (LeonardoMolleker) [reply lib/src/pages/dice_page.dart]: No, it's not working, it keeps same

Comment (FranciscoGaraventa) [reply lib/src/pages/dice_page.dart]: Try to put ',' after ')', on the line 44.

Comment (LeonardoMolleker) [reply lib/src/pages/dice_page.dart]: Now yeah. Do i the push again?

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM!

**Pull request merged**